blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -99,6 +99,12 @@ Changes
 - Added support of using units inside byte-size or time bases statement
   parameters values. E.g. '1mb' for 1 MegaByte or '1s' for 1 Second.
 
+- Changed hdfs repository parameter, ``user``, to ``security.principal`` so
```

Comment:
also, super minor, but we generally add the newer changes at the top. can you please move them just above "Added support for CrateDB license management ..."